draft-birkholz-scitt-software-use-cases.md

```diff
@@ -167,6 +167,35 @@ There is no standardized way to:
 * to check for the trustworthiness of a provider
 * to check the integrity of modifications or transformations done by a provider
 
+## Verifying Firmware Provenance for large and complex systems
+
+In contrast to operating systems or user space software components of a large and complex systems, firmware components are often already executed during boot-cycles before there is an opportunity to authenticate them.
+
+Authentication takes place, for example, by checking an accompanying signature or by creating a hash of measurements and comparing them to a reference value.
+Corresponding procedures are often called authenticated, measured, or secure boot. The output of these high assurance boot procedures is often used as input to more complex remote attestation procedures.
+
+If measurements before execution are not possible, static after-the-fact analysis is required.
+
+In these cases measurements (e.g., a hash or digests) are stored in a protected or shielded environment (e.g., TEEs or TPMs) on these systems itself.
+
+After finishing a high assurance boot sequence, these measurements about foundational firmware are retrieved after-the-fact from shielded locations and must be compared to reference values that are part of Reference Integrity Measurements (RIMs).
```

Comment:
Yogesh pointed out to me that this could maybe mean Reference Integrity Manifest (which would be less confusing, but maybe not what we wanted to say)?

Reply:
I think we mean Manifest!